scripts/defaultBuild.sh

```diff
@@ -33,8 +11,9 @@ defaultBuild()
     echo
     echo Build step
     echo
-    pushd $BUILDDIR                             || exit 1
-    cmake $CMAKE_OPTIONS ..                     || exit 1
+    local SRCDIR=$PWD
+    pushd $BUILDDIR || exit 1
+    cmake -C $SRCDIR/scripts/defaultCIBuild.cmake $EXTRA_CMAKE_ARGS .. || exit 1
```

Comment:
`$EXTRA_CMAKE_ARGS` should come before the cache cmake. This will allow default CUDA architecture configuration, with it being customizable through cluster scripts.

Reply:
I don't know about that - the extra cmake args come after the initial cache script *so that* they override whatever is set in that cmake cache script

Reply:
The argument here was more for clarity I suppose. When building with the current CMake cache, CMake shows that you are using the default architecture 70, and then under the hood re-sets it to whatever you have overridden it to. If this behaviour is OK, then I don't have a problem, but I liked having the arch information printed with the cache build when running with my suggestion.